newIDE/app/src/stories/componentStories/UserChip/UserChip.stories.js

```diff
@@ -6,22 +6,43 @@ import { action } from '@storybook/addon-actions';
 import muiDecorator from '../../ThemeDecorator';
 import paperDecorator from '../../PaperDecorator';
 
-import UserChip from '../../../UI/User/UserChip';
+import UserChipComponent from '../../../UI/User/UserChip';
 import { indieUserProfile } from '../../../fixtures/GDevelopServicesTestData';
+import { type Profile } from '../../../Utils/GDevelopServices/Authentication';
 
 export default {
   title: 'User chips/UserChip',
-  component: UserChip,
+  component: UserChipComponent,
   decorators: [paperDecorator, muiDecorator],
+  argTypes: {
+    withNotifications: {
+      name: 'With notifications',
+      type: { name: 'boolean', required: true },
+      defaultValue: false,
+    },
+    profile: {
+      name: 'User profile',
+      control: { type: 'radio' },
+      options: ['Signed in', 'Unsigned'],
```

Comment:
uberpitcking: not sure if Unsigned is proper english, maybe "Not signed in" (https://english.stackexchange.com/questions/251800/a-word-for-a-non-logged-in-user)
(anonymous could work too)